task_processing/plugins/mesos/retrying_executor.py

```diff
@@ -91,8 +106,8 @@ def run(self, task_config):
         if task_config.task_id not in self.task_retries:
             with self.task_retries_lock:
                 self.task_retries = self.task_retries.set(
-                    task_config.task_id, self.retries)
-        self.executor.run(task_config)
+                    task_config.task_id, 1)
+        self.executor.run(self._get_encoded_task_config(task_config))
```

Comment:
and one more function name nitpick: `task_config_with_retry` ?

Reply:
Fixed.